torch/csrc/lazy/ts_backend/ts_node.cpp

```diff
@@ -18,17 +18,11 @@ namespace torch {
 namespace lazy {
 
 const Shape& GetShapeFromTsOutput(const Output& output) {
-  if (auto* tsnode = dynamic_cast<const TsNode*>(output.node)) {
-    return tsnode->shape(output.index);
-  }
-  throw std::runtime_error("Expected TsNode but could not dynamic cast");
+  return output.node->shape(output.index);
```

Comment:
Can't this just be
```suggestion
  return output.shape();
```
also, is this function even used anymore? Can it be removed?

Reply:
In fact, it looks like you removed it from `ts_node.h` altogether. So, I think this definition can be removed